src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js

```diff
@@ -64,8 +64,9 @@ const UniqueTokenExpandedStateContent = ({
     if (asset.image_url?.startsWith?.(GOOGLE_USER_CONTENT_URL) && size > 0) {
       return `${asset.image_url}=w${size * MAX_IMAGE_SCALE}`;
     }
+    if (asset?.isPoap) return asset.animation_url;
     return asset.image_url;
-  }, [asset.image_url, size]);
+  }, [asset.animation_url, asset.image_url, asset?.isPoap, size]);
```

Comment:
why `asset?.isPoap` if no one else is checking ?

Reply:
funky, approve again pls 😘